packages/mattermost-redux/src/selectors/entities/channel_categories.ts

```diff
@@ -468,14 +469,32 @@ export function makeSortChannels() {
     };
 }
 
-export function makeGetChannelsForCategory() {
+
+export function makeGetChannelIdsForCategory() {
```

Comment:
can a plugin be using this? if so, we should mark it as a breaking change at least.

Reply:
They could be but my guess is it's unlikely. How do we typically handle changing selectors?

I suppose I could leave the old selectors in instead of replacing them

Reply:
I don't think I've done it before, maybe @devinbinnie or @hmhealey knows

Reply:
Technically, plugins are still using the `mattermost-redux` repository, so this won't break them at present. I doubt anyone is using this anyway.